src/lib/sw_engine/tvgSwCommon.h

```diff
@@ -274,7 +274,7 @@ static inline uint32_t COLOR_INTERPOLATE(uint32_t c1, uint32_t a1, uint32_t c2,
 
 static inline uint8_t ALPHA_MULTIPLY(uint32_t c, uint32_t a)
 {
-    return (c * a) >> 8;
+    return ((c * a + 0xff) >> 8);
```

Comment:
we had this discussion already in #230 
result from this formula is 1 higher the the exact result in about 30% of the cases (out of all over 65k)
so it's not really the 'precise alpha multiplying', but the solution that I wanted to apply, has one additional multiplication. 

the question is how precise we want to be and what's the price that we are ready to pay.

Reply:
@mgrudzinska i know. i decided this since it's better precise than current version.
Most cases it results in correct result,  only when values c= 0 a=0  results in 1.
but we don't allow alpha == 0 case in the rasterization.

Reply:
not only for c=a=0, for example c=a=1 in this formula gives also 1 instead of 0, but I understand your point. you are right that it's better than the current version

Reply:
c=a=1
1 * 1 + 255 / 256 = 1  (correct result.)

Reply:
@mgrudzinska yeah see if it's actual issued later.